server/store/user.go

```diff
@@ -135,7 +135,7 @@ func (s *pluginStore) DeleteUser(mattermostUserID string, server string, userNam
 
 func (s *pluginStore) loadUser(mattermostUserID string) (*user, error) {
 	u := &user{}
-	err := LoadGOB(s.userStore, fmt.Sprintf("%s%s", UserStoreKeyPrefix, mattermostUserID), u)
+	err := s.userStore.loadGOB(fmt.Sprintf("%s%s", UserStoreKeyPrefix, mattermostUserID), u)
```

Comment:
Seems weird that we're using `JSON` and `GOB`. Does it make sense to use only one? @hanzei

Reply:
@sibasankarnayak Unless there is something specific to splunk that requires us to use `GOB` (which that wouldn't really make sense because it's just in our store layer), let's only use `JSON` since that's what is used in all other plugins.

Reply:
@mickmister sure , used only JSON